packages/panels/resources/lang/nl/layout.php

```diff
@@ -52,4 +52,10 @@
 
     ],
 
+    'top_bar' => [
+
+        'avatar' => 'Avatar vertegenwoordigt :user',
+
+    ],
```

Comment:
Please remove new lines in arrays when there is no nesting, as per the other instances in these files